### PR TITLE
ZH-trans: add update-intro.html

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            <div class="col-md-8">
+            <!--
+            <h3>Updating an application</h3>
+            -->
+            <h3>更新应用程序</h3>
+            
+            <!--
+            <p>Users expect applications to be available all the time and developers are expected to deploy new versions of them several times a day. In Kubernetes this is done with rolling updates. <b>Rolling updates</b> allow Deployments' update to take place with zero downtime by incrementally updating Pods instances with new ones. The new Pods will be scheduled on Nodes with available resources.</p>
+            -->
+            <p>用户希望应用程序始终可用，并且开发人员应该每天多次部署新版本的应用程序。在 Kubernetes 中，通过滚动更新完成的。 <b>滚动更新</b> 允许通过使用新的实例逐步更新 Pod 实例，零停机进行部署更新。新的 Pod 将在具有可用资源的节点上进行调度。</p>
```
Comment (tengqm) [inline content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html]: > 在 Kubernetes 中，通过滚动更新完成的。

这句话不通。

Comment (tengqm) [inline content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html]: > 零停机进行部署更新

这里的 Deployment 为什么要翻译？

File: content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+            
+            <!--
+            <p>In the previous module we scaled our application to run multiple instances. This is a requirement for performing updates without affecting application availability. By default, the maximum number of Pods that can be unavailable during the update and the maximum number of new Pods that can be created, is one. Both options can be configured to either numbers or percentages (of Pods).
+            In Kubernetes, updates are versioned and any Deployment update can be reverted to previous (stable) version.</p>
+            -->
+            <p>在前面的模块中，我们将应用程序扩展为运行多个实例。这是在不影响应用程序可用性的情况下执行更新的要求。默认情况下，更新期间可用的最大 pod 数和可以创建的 pod 数都是 1。这两个选项都可以配置为（pod）数字或百分比。
```
Comment (tengqm) [inline content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html]: > 更新期间可用的最大 pod 数

仔细读原文！

File: content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+            -->
+                <p> 在下面的交互式教程中，我们将应用程序更新为新版本，并执行回滚。</p>
+            
+            </div>
+        </div>
+        <br>
+
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/update/update-interactive/" role="button">Start Interactive Tutorial <span class="btn__next">›</span></a>
```
Comment (tengqm) [inline content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html]: > Start Interactive Tutorial

漏翻

File: content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html (hunk 2 above)
Comment (xichengliudui) [reply content/zh/docs/tutorials/kubernetes-basics/update/update-intro.html]: 这里是我手误了。感谢！

**Comment by fleeto** [comment]
`[trans-bot:merged]`

Thank you @xichengliudui, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/604) had been updated.

**Pull request merged**